### PR TITLE
Update Sentry DSN & fix Sentry integration

### PR DESCRIPTION
**Pull request by jairideout**
## Status
- [x] **Ready**

## Description
Closes DEV-7183

## Related PRs
- [x] This PR is independent

### CODE REVIEW

**Review by alenajk — APPROVED** [review]

File: onecodex/utils.py [hunk 1]
```diff
@@ -242,36 +241,64 @@ def custom_exc(shell, etype, evalue, tb, tb_offset=None):
...
                     if x
                 ]
             sentry_sdk.init(
-                dsn=os.environ.get("ONE_CODEX_SENTRY_DSN", default_dsn),
-                install_sys_hook=install_sys_hook,
-                raise_send_errors=False,
-                ignore_exceptions=ignore_exceptions,
-                include_paths=[__name__.split(".", 1)[0]],
+                dsn=os.environ.get(
+                    "ONE_CODEX_SENTRY_DSN",
```
Comment (Keats) [inline onecodex/utils.py:295]: When do we want to override that? For localhost?

File: onecodex/utils.py [hunk 2]
```diff
@@ -244,9 +243,6 @@ def custom_exc(shell, etype, evalue, tb, tb_offset=None):
 
 def init_sentry(user_context=None, extra_context=None):
     if os.environ.get("ONE_CODEX_NO_TELEMETRY") is None:
-        default_dsn = base64.b64decode(
```
[the remark this answers is not in the document]
Comment (Keats) [reply onecodex/utils.py:247]: Maybe so some systems can't detect the telemetry URL and block the package? I don't know.

File: onecodex/utils.py (hunk 1 above)
Comment (pkonieczny) [reply onecodex/utils.py:295]: We override this in notebooks

**Review by pkonieczny — APPROVED** [review]
LGTM 👍

**Pull request merged**